scripts/windows/build.py

```diff
@@ -29,7 +29,7 @@ def install(self):
     if self.check() is False:
       return
 
-    os.chdir('../..')
+    os.chdir('..')
```

Comment:
i would add at least one comment  about where this goes

Reply:
Noted! I'll create a trello task for that.